ui/src/app/components/direct-connection/direct-connection.component.scss

```diff
@@ -0,0 +1,17 @@
+.container {
```

Comment:
General comment on the SCSS files. We should probably use a global style rather than having custom styles per component. Any reason why this is required ?

Reply:
The styles within the file are local to the component only and do not overwrite other styles. In ideal condition we can then position the component anywhere in the page having the same style. So in a way, state, behaviour and style stay together.

Reply:
As discussed, we will use global CSS for common styles and for new components